procedures/accounts_setup.rst

```diff
@@ -0,0 +1,19 @@
+Here is a list of accounts and access needed to be successful in Release Engineering
+
+General:
+ * `JIRA <https://jira.mozilla.com/projects/RELENG/issues>`_ - add username to Mozilla instance
+ * `Commit Access <https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/>`_ - fill out form, file bug
+ * Bugzilla
+ * Phabricator
+ * Github
+ * `Mozilla VPN <https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=30769829>`_
+
+Release Duty:
+ * ?
+
+Mailing Lists:
+ * 
+
+LDAP Permissions:
+ * 
```

Comment:
Oh, good point! Should we point a href to https://bugzilla.mozilla.org/show_bug.cgi?id=1681190 to begin with and refine later?